app/script/view_model/content/InputBarViewModel.js

```diff
@@ -227,6 +203,14 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       return conversationEntity.localMessageTimer() && !conversationEntity.hasGlobalMessageTimer();
     });
 
+    this.conversationEntity.subscribe(this.loadInitialStateForConversation.bind(this));
+    this.draftMessage.subscribe(message => {
+      if (!this.conversationEntity()) {
+        return;
+      }
+      this._saveDraftState(this.conversationEntity(), message.text, message.mentions);
```

Comment:
Not a favor of an early return on the negative case on this one. Positive approach is better as that is what we are looking for and not for what is not doing:

````
 if (this.conversationEntity()) {
  this._saveDraftState(this.conversationEntity(), message.text, message.mentions);
}
````